Userland/Applications/NetworkSettings/NetworkSettingsWidget.cpp

```diff
@@ -1,23 +1,28 @@
 /*
  * Copyright (c) 2022, Maciej <sppmacd@pm.me>
+ * Copyright (c) 2023, Fabian Dellwing <fabian@dellwing.net>
  *
  * SPDX-License-Identifier: BSD-2-Clause
  */
 
 #include "NetworkSettingsWidget.h"
-
 #include <AK/DeprecatedString.h>
 #include <AK/IPv4Address.h>
+#include <AK/JsonObject.h>
 #include <AK/JsonParser.h>
 #include <Applications/NetworkSettings/NetworkSettingsGML.h>
+#include <Kernel/API/POSIX/fcntl.h>
```

Comment:
Hmm Kernel includes in userland applications are really suspicious.
Can't you use `<fcntl.h>` directly?